common/static/sass/_mixins.scss

```diff
@@ -276,10 +276,10 @@
   @extend %t-action4;
   text-align: center;
   border-radius: ($baseline/4);
-  border: 1px solid $blue-l2;
+  //border: 1px solid $blue-l2;
```

Comment:
Author To-Do: This file's changes should be reverted, as it was unintentionally included from earlier debugging of a sass compilation issue on master.